src/pages/cropper.astro

```diff
@@ -73,7 +68,7 @@ import Base from "../components/base.astro";
 		display: block;
 	}
 
-	button {
+	/* button {
```

Comment:
I think we can just remove these parts. Don't foresee us coming back to these styles.

Reply:
Okay